contentqa/app/controllers/contentqa/compare_controller.rb

```diff
@@ -9,8 +9,14 @@ def index
       @id = params[:id]
 
       doc = @id ? item_fetch(@id)['docs'].first : get_random_doc(fetch['count'])
-      
-      @original = JSON.pretty_generate(doc['originalRecord'] || {})
+      @original =
+          if !doc['originalRecord']
```

Comment:
I _think_ that we are supposed to use `.empty?` for cases like this in Ruby. Without looking it up and refreshing my memory, I think there may be subtleties in the evaluation of this as a boolean.